### PR TITLE
chore(templates/website): adds seeded users to readme

### PR DESCRIPTION
**Pull request by calvincchong**
## Description
All other references to seed accounts and passwords were in the Auth example: 

[Auth Example](https://github.com/payloadcms/payload/edit/master/examples/auth/payload/README.md
)

However, the valid username and password is slightly different for the template website page. This information would be very helpful in the README of each template installation as I could not find this info easily within the Payload Docs.

Additional note: the e-commerce site seed data may need to be updated as well.

- [ X ] I have read and understand the [CONTRIBUTING.md](../CONTRIBUTING.md) document in this repository.

## Type of change

- [ X ] Chore (non-breaking change which does not add functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Change to the [templates](../templates/) directory (does not affect core functionality)
- [ ] Change to the [examples](../examples/) directory (does not affect core functionality)
- [ ] This change requires a documentation update

## Checklist: N / A

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ X ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: templates/website/README.md [hunk 1]
```diff
@@ -227,6 +227,10 @@ That's it! The Docker instance will help you get up and running quickly while al
 
 To seed the database with a few pages, posts, and projects you can run `yarn seed`. This template also comes with a `GET /api/seed` endpoint you can use to seed the database from the admin panel.
 
+On boot, a seed script is included to create 2 users:
```
Comment (jacobsfletch) [inline templates/website/README.md:230]:
```suggestion
The seed script will also create two users for demonstration purposes only:
```

File: templates/website/README.md [hunk 2]
```diff
@@ -227,6 +227,10 @@ That's it! The Docker instance will help you get up and running quickly while al
...
+1. email `demo-author@payloadcms.com`, password `password`, the role `admin`.
```
Comment (jacobsfletch) [inline templates/website/README.md:231]:
```suggestion
1. Demo Author
    - Email: `demo-author@payloadcms.com`
    - Password: `password`
    - Role: `admin`
```

File: templates/website/README.md [hunk 3]
```diff
@@ -227,6 +227,10 @@ That's it! The Docker instance will help you get up and running quickly while al
...
+2. email `demo-user@payloadcms.com`, password `password`, the role `user`
```
Comment (jacobsfletch) [inline templates/website/README.md:232]:
```suggestion
2. Demo User
    - Email: `demo-user@payloadcms.com`
    - Password: `password`
    - Role: `user`
```

**Comment by jacobsfletch** [comment]
@calvincchong this is great! I made some adjustments to syntax. One other thing to note is that the templates do not auto-seed like the examples do, so the description of `On boot, a seed script is included to create 2 users:` was not quite accurate. Templates require _you_ to seed manually, and it is also not intended to seed your initial admin user for you.

**Review by calvincchong — COMMENTED** [review]
@jacobsfletch Thanks for taking a look at this! Moving from Strapi, I had just started testing out the different versions of the examples and forgot that the templates is manually seeded. Removing "On boot" makes sense!

**Comment by jacobsfletch** [comment]
Merged. Thanks @calvincchong!

**Pull request merged**